packages/block-library/src/file/block.json

```diff
@@ -34,6 +34,13 @@
 			"type": "string",
 			"source": "html",
 			"selector": "a[download]"
+		},
+		"displayPreview": {
+			"type": "boolean"
+		},
+		"previewHeight": {
+			"type": "number",
+			"default": 800
```

Comment:
I think this is pretty big for default 😄 @jasmussen any input with a design perspective?

Reply:
That is a bit big. But not insanely big. I'm mostly thinking in terms of what if you inserted this inside a column meant to not be that tall, or if you start on mobile. I think just changing to something like 600 might strike a fine default which we can always revisit.

And if/when we do revisit, I'd love to be able to set VH units as a default! Or perhaps even assign an `aspect-ratio` so we don't need to provide a height (though the support would need to be much better https://caniuse.com/mdn-css_properties_aspect-ratio)

By the way, I endorse the name of this branch!

Reply:
👍🏻 6a4de29076